codegen/src/main/java/software/amazon/awssdk/codegen/poet/builder/BaseClientBuilderClass.java

```diff
@@ -741,6 +750,33 @@ private MethodSpec invokePluginsMethod() {
         return builder.build();
     }
 
+    private MethodSpec getSdkPluginsMethod() {
+
+        ParameterizedTypeName parameterizedTypeName = ParameterizedTypeName
+                .get(ClassName.get(List.class), ClassName.get(SdkPlugin.class));
+
+        MethodSpec.Builder builder = MethodSpec.methodBuilder("getSdkPlugins")
+                .addModifiers(PROTECTED)
```

Comment:
Does this method need to be `protected`, or is `private` sufficient?

Reply:
Good catch, private is sufficient.